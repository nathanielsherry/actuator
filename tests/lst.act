#!/usr/local/bin/act

flow A from "a,b,c,d" via split[","]|lst.reverse|lst.join[", "] to @agg;

flow B from "a,b,c,d" via split[","]|lst.tail|lst.head to @agg;

flow C from "1,2,3,4" via split[","]|lst.ints|lst.sum to @agg;
flow D from "1,2,3,4" via split[","]|lst.ints|lst.max to @agg;
flow E from "1,2,3,4" via split[","]|lst.ints|lst.min to @agg;
flow F from "1,2,3,4" via split[","]|lst.ints|lst.slice[1,3] to @agg;

flow agg from inflows;
