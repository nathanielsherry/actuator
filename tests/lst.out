['d, c, b, a', 'b', 10, 4, 1, [2, 3]]
